### ISSUE TITLE
BUG: No "Destroy Documentation" for package after add, doc, remove, re-add

### ISSUE THREAD

**Issue by larsoner** [pypa/warehouse#5641]
I along with @Titan-C are maintainers of the [sphinx-gallery](https://pypi.org/project/sphinx-gallery/) package. Long ago (apparently) the package was named `sphinxgallery` (possibly maintained by @Titan-C), but apparently it:

1. Currently exists as a package at https://pypi.org/project/sphinxgallery/, even though @Titan-C thinks he removed it, and
2. There are minimal docs available at https://pythonhosted.org/sphinxgallery/ that people have found and been confused by

I'm not exactly sure what the situation is here (name squatting?) or if PEP541 is the right PEP, but it would be good if we could fix it. FWIW the maintainer of `sphinxgallery` maintains 1153 (!) packages, most of which seem to be hyphen-less versions of real packages, such as [FlaskLogin](https://pypi.org/project/FlaskLogin/) package that mimics [Flask-Login](https://pypi.org/project/Flask-Login/#history). So I'm not sure what's going on. It looks related to at least #4413, #4629, and #1506.

My PyPi username is Eric89GXL.

**Comment by jamadden** [comment]
The `sphinxgallery` project describes itself as "A package to prevent exploit", and its `setup.py` (as well as its `__init__.py` is full of things like `raise Exception("You probably meant to install and run sphinx-gallery")`. So it would seem that the author is attempting some sort of white-hat squatting.

The Author email on the PyPI page for the project matches the email we have registered to the owner of the package. I would suggest emailing them and seeing of they will transfer the project back.

**Comment by larsoner** [comment]
Argh I did look at the source briefly but obviously not thoroughly enough. Sorry for the noise.

**Comment by larsoner** [comment]
I was added as an Owner of the `sphinxgallery` repo, but there is no option to get to the docs to "Destroy Documentation":

![Screenshot from 2019-03-29 14-00-38](https://user-images.githubusercontent.com/2365790/55253189-115eaf00-522b-11e9-9762-7d944ef59112.png)

Following the URL scheme from other repos that do have docs, if I force it to go to the "Documentation" page there is no button there:

![Screenshot from 2019-03-29 14-01-38](https://user-images.githubusercontent.com/2365790/55253267-423ee400-522b-11e9-9dcb-aab444d0ea80.png)


This is probably specific to this situation, where:

1. `sphinxgallery` package was added
2. docs were uploaded
3. `sphinxgallery` package was deleted (without destroying docs)
4. `sphinxgallery` package was re-added (with no docs, and thus no reference to docs)

I tried just pushing a redirect `index.html`, hoping that that would "notify" PyPi that there actually were docs, and then it would allow me to delete them (or if it didn't, at least the redirect would work), but as expected:
```
larsoner@bunk:~/python/sphinxgallery-0.1.1$ python2 setup.py upload_sphinx
running upload_sphinx
Submitting documentation to https://upload.pypi.org/legacy/
Upload failed (410): Uploading documentation is no longer supported, we recommend using https://readthedocs.org/.
```
So I think this situation will require some manual intervention at the PyPi end to remove the old docs.
